app/src/androidTest/java/org/mozilla/fenix/ui/SettingsSearchTest.kt

```diff
@@ -376,14 +376,38 @@ class SettingsSearchTest {
         }
     }
 
-    @Ignore("Test failure caused by: https://bugzilla.mozilla.org/show_bug.cgi?id=1807298")
     // Expected for en-us defaults
     @Test
     fun deleteAllSearchEnginesTest() {
         homeScreen {
         }.openThreeDotMenu {
         }.openSettings {
         }.openSearchSubMenu {
+            /*
+            When unified search will be enabled by default,
```

Comment:
Is there a preference we can check to see if it's enabled?

Reply:
Yes, we have it [here](https://github.com/mozilla-mobile/fenix/blob/main/app/src/main/java/org/mozilla/fenix/utils/Settings.kt#L1468), seems to be controlled by nimbus and is behind a [feature flag](https://github.com/mozilla-mobile/fenix/blob/main/app/src/main/java/org/mozilla/fenix/FeatureFlags.kt#L53)

Reply:
Maybe you can use the method defined in my PR https://github.com/mozilla-mobile/fenix/pull/28673/files#diff-663a082814e9b214da9c34f4ea9afef9c74a553b356166681e36ce8e7275b5a4R474